### DISCUSSION TITLE
application unable to process

### DISCUSSION THREAD

**Discussion by namprice227** [community/community#66572] [GitHub Education]
### Hi everyone,

I am currently a university student and I want to join github global campus. However, I have tried a lot of time and whenever i press proceed my application button, it will always load and redirect to Upload proof of your academic status page. I have keyed in my school email and check everything. I hope that you can resolve this problem for me

**Comment by asd58584388** [comment]
Me too, after submitting the application, there is no feedback

**Reply by Ritik82** [reply]
Same issue

**Reply by yippp** [reply]
Same issue
